x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel.tsx

```diff
@@ -205,8 +205,7 @@ export const InnerWorkspacePanel = React.memo(function InnerWorkspacePanel({
           eventName: 'lensVisualizationRenderTime',
           duration: currentTime - visualizationRenderStartTime.current,
           key1: 'time_to_data',
-          value1:
-            dataReceivedTime.current - visualizationRenderStartTime.current - esTookTime.current,
+          value1: dataReceivedTime.current - visualizationRenderStartTime.current,
```

Comment:
https://github.com/elastic/kibana/pull/192245 changed the meaning of this `time_to_data`. I wouldn't change it, since it changes how to interpret that metric in historical comparisons. If we really need the difference, we can compute it at runtime (runtime fields, esql, ...)